spyder_terminal/widgets/terminalgui.py

```diff
@@ -250,6 +246,28 @@ def __init__(self, parent, CONF, term_url='http://127.0.0.1:8070',
 
         self.initial_y_pos = 0
         self.setFocusPolicy(Qt.ClickFocus)
+        self.setup()
+
+    def setup(self):
+        """Create the terminal shortcuts."""
```

Comment:
This docstring seems incorrect. Should it be

    Create the terminal context menu

?